### PR TITLE
Update terraform version to 0.11.0

### PR DESCRIPTION
**Pull request by JoseThen**

### CODE REVIEW

File: rt/version.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 // The following will be filled in by the compiler
 var GitCommit string
 
-const TerraformVersion = "0.10.8"
+const TerraformVersion = "0.11.0"
 const Version = "0.10.1"
```
Comment (cjmurphy42) [inline rt/version.go]: we will want to bump this to version 0.11.0 as well

File: rt/version.go [hunk 2]
```diff
@@ -12,7 +12,7 @@ import (
 // The following will be filled in by the compiler
 var GitCommit string
 
-const TerraformVersion = "0.10.8"
+const TerraformVersion = "0.11.0"
```
Comment (cjmurphy42) [inline rt/version.go]: any reason not to goto 0.11.14?

Comment (JoseThen) [reply rt/version.go]: I was just following semver versioning here. Since we are not really fixing any bugs and just increasing the minor version.

Comment (JoseThen) [reply rt/version.go]: I see what you mean now, updates pushed here: 
8ec8dda

File: rt/version.go (hunk 1 above)
Comment (JoseThen) [reply rt/version.go]: sounds good, update pushed: 
9f57e21

**Pull request merged**